docs/deployment/uwsgi_cookiecutter_2__emperor.rst

```diff
@@ -57,6 +57,14 @@ WSGI entry point.
     to allow logging within your application. See
     :ref:`pyramid:logging_config`.
 
+#.  Create a directory for your project's log files, and set ownership on the
+    directory
+
+    .. code-block:: bash
+
+        $ cd /var/log
+        $ sudo mkdir uwsgi
+        $ sudo chown ubuntu:www-data uwsgi
```

Comment:
Ran through both parts of the tutorial on a fresh EC2 machine today, which surfaced a couple technical changes. The first is that these directories are needed.